src/Pillar-Tests-ExporterCore.package/PRDocumentListTransformerTest.class/instance/testAnnotationWithEmptyMultipleMustacheTemplateShouldNotContainsRaw.st

```diff
@@ -0,0 +1,18 @@
+private
+testAnnotationWithEmptyMultipleMustacheTemplateShouldNotContainsRaw
```

Comment:
I'd add a test comment showing the annotation we are creating. Something like:

"${documentList:....}"

Like that we understand what is expected more clrearly and we don't need to mentally parse the creation of the annotation.